tests/stub.rs

```diff
@@ -0,0 +1,6 @@
+#[cfg(loom)]
+/// Run loom tests with something like
+/// ```
+/// $ RUSTFLAGS="--cfg loom" cargo test --release loom
+/// ```
+mod loom;
```

Comment:
Could the three loom test files be collapsed into this file or is there a particular reason for this setup?

Reply:
I figured it would nice for organizing a lot of tests that all get lumped under loom, but there probably won't end up being enough loom tests for that to make sense :sweat_smile: